src/axom/quest/MarchingCubes.cpp

```diff
@@ -206,31 +206,39 @@ void MarchingCubesSingleDomain::allocateImpl()
   if(m_runtimePolicy == RuntimePolicy::seq)
   {
     m_impl = m_ndim == 2
-      ? std::unique_ptr<ImplBase>(new MarchingCubesImpl<2, axom::SEQ_EXEC>)
-      : std::unique_ptr<ImplBase>(new MarchingCubesImpl<3, axom::SEQ_EXEC>);
+      ? std::unique_ptr<ImplBase>(
+          new MarchingCubesImpl<2, axom::SEQ_EXEC, RAJA::seq_exec>)
+      : std::unique_ptr<ImplBase>(
+          new MarchingCubesImpl<3, axom::SEQ_EXEC, RAJA::seq_exec>);
   }
 #ifdef _AXOM_MC_USE_OPENMP
   else if(m_runtimePolicy == RuntimePolicy::omp)
   {
     m_impl = m_ndim == 2
-      ? std::unique_ptr<ImplBase>(new MarchingCubesImpl<2, axom::OMP_EXEC>)
-      : std::unique_ptr<ImplBase>(new MarchingCubesImpl<3, axom::OMP_EXEC>);
+      ? std::unique_ptr<ImplBase>(
+          new MarchingCubesImpl<2, axom::OMP_EXEC, RAJA::seq_exec>)
```

Comment:
`RAJA::seq_exec` is the serial loop policy for `axom::OMP_EXEC`.  Both work with host memory.

Reply:
This should be axom::SEQ_EXEC, right?

Reply:
Yes.  I changed the code to use the Axom typedefs instead of the RAJA types.